src/bin/railroad.rs

```diff
@@ -0,0 +1,47 @@
+extern crate railroad_dsl;
+extern crate structopt;
+
+use std::error::Error;
+use std::fs;
+use std::path::PathBuf;
+use std::io::{self, Read};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
```

Comment:
This puts us into `nightly`-winter-wonderland, doesn't it? I'm fine with that as long as `railroad` stays on stable. I'll add something to the readme then.

Reply:
Right - I'll see if I can find a variant that works without warnings on nightly and stable.